dunereco/DUNEWireCell/dune10kt-hd/params.jsonnet

```diff
@@ -0,0 +1,182 @@
+        // match what field response functions also used.
+        response_plane: 10*wc.cm, // relative to collection wires
+        local res_plane = 0.5*apa_w2w + self.response_plane,
+
+        // The cathode plane is like the anode cut off plane.  Any
+        // depo not between the two is dropped prior to drifting.
+        local cpa_plane = apa_cpa - 0.5*cpa_thick,
+
+
+        // The volumes are then defined in terms of these above
+        // numbers.  You can use "wirecell-util wires-info" or
+        // "wirecell-util wires-volumes" or others to understand the
+        // mapping of anode number to the 6 locations in X and Z.  For
+        // Larsoft wires the numbering is column major starting at
+        // small X and Z so the centerline is -/+/-/+/-/+.  Also
+        // important is that the faces are listed "front" first.
+        // Front is the one with the more positive X coordinates and
+        // if we want to ignore a face it is made null.
+        volumes: [
+            {
+                local sign = 2*(n%2)-1,
+                local centerline = sign*apa_cpa,
+                wires: n,       // anode number
+                name: "apa%d"%n,
+                faces:
+                // top, front face is against cryo wall
+                if sign > 0
+                then [
+                    {
+                        anode: centerline + apa_plane,
+                        response: centerline + res_plane,
+                        cathode: centerline + cpa_plane, 
+                    },
+                    {
+                        anode: centerline - apa_plane,
+                        response: centerline - res_plane,
+                        cathode: centerline - cpa_plane, 
+                    }
+                ]
+                // bottom, back face is against cryo wall
+                else [
+                    {
+                        anode: centerline + apa_plane,
+                        response: centerline + res_plane,
+                        cathode: centerline + cpa_plane, 
+                    },
+                    {
+                        anode: centerline - apa_plane,
+                        response: centerline - res_plane,
+                        cathode: centerline - cpa_plane, 
+                    }
+                ],
+            } for n in std.range(0,11)], // twelve anodes
+
+        // This describes some rough, overall bounding box.  It's not
+        // directly needed but can be useful on the Jsonnet side, for
+        // example when defining some simple kinematics.  It is
+        // represented by a ray going from extreme corners of a
+        // rectangular solid.  Again "wirecell-util wires-info" helps
+        // to choose something.
+        bounds : {
+          tail: wc.point(-363.376, -600.019, -0.87625, wc.cm),
+          head: wc.point( 363.376,  600.019, 1393.46, wc.cm),
+        }
+    },
+
+    daq: super.daq {
+        nticks: 6000,
```

Comment:
Does this get overwritten later on?  If we change the number of ticks in LArSoft will thie line supersede the larsoft value?

Reply:
Is this still under review?  I think we have a general issue with LArSoft plugins that do not share common configurations.  There is a manual step needed to align the configurations across domains owned by different software components.  We had a discussion about this in the future frameworks requirements workshop, but I don't think we had a good solution.

Reply:
I've pinged Haiwang to take a look.
We've had some success in combatting this in FD, by linking the params between config with `@local`.  That obviously only works when both configs are fcl-based.